coalib/output/ConfWriter.py

```diff
@@ -16,7 +16,8 @@ def __init__(self,
                  section_name_surroundings=None,
                  section_override_delimiters=(".",),
                  unsavable_keys=("save",)):
-        section_name_surroundings = section_name_surroundings or {"[": "]"}
+
+ 	section_name_surroundings = {"[": "]"} if section_name_surroundings is None else section_name_surroundings
```

Comment:
It looks like you've removed a space from the front of the line (which messes up indentation everywhere)